src/Compilers/CSharp/Test/Semantic/Semantics/PropertyFieldKeywordTests.cs

```diff
@@ -0,0 +1,1246 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+#nullable disable
+
+using System.Linq;
+using Microsoft.CodeAnalysis.CSharp.Symbols;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+using Microsoft.CodeAnalysis.CSharp.Test.Utilities;
+using Microsoft.CodeAnalysis.CSharp.UnitTests;
+using Roslyn.Test.Utilities;
+using Xunit;
+
+namespace Microsoft.CodeAnalysis.CSharp.Semantic.UnitTests.Semantics
+{
+    /* PROTOTYPE(semi-auto-props):
+     * Add tests for field attribute target, specially for setter-only
+     * nameof(field) should work only when there is a symbol called "field" in scope.
+     * nullability tests, ie, make sure we get null dereference warnings when needed.
+     */
+
+    /* PROTOTYPE(semi-auto-props):
+     * Regarding AbstractFlowPass and DefiniteAssignment, it looks like the logic there requires an adjustment since the behavior of IsAutoPropertyWithGetAccessor has changed.
+     * We can no longer guarantee that accessing the property under the checked conditions is always equivalent to accessing a backing field.
+     * For example, if the corresponding accessor has a body, we don't really know what it is doing.
+     * Also, it looks like usages of AbstractFlowPass.RegularPropertyAccess helper should be reviewed and appropriate adjustments should be made according to it's behavior change.
+     */
+
+    // PROTOTYPE(semi-auto-props): All tests should verify that we do not synthesize fields unexpectedly.
+    // Success cases can do this by observing emitted metadata.
+    // Note that private symbols are filtered out by default, either type IL should be checked, or
+    // compilation options should be adjusted to disable the filtering.
+    // Error cases should check GetFieldToEmit after checking diagnostics, but before checking NumberOfPerformedAccessorBinding.
+
+    // PROTOTYPE(semi-auto-props): Speculative semantic model tests. Speculating with a field keyword within an accessor of a regular property then emitting shouldn't emit the backing field.
+
+    // PROTOTYPE(semi-auto-props): Add tests for GetMembers API. It shouldn't return the backing field of field keyword, but returns it for initializers even if field keyword exists.
```

Comment:
>but returns it for initializers even if field keyword exists. [](http://example.com/codeflow?start=121&length=61)

I do not think this is accurate. We never add ```field``` keyword backing field to the members list. We intentionally changed behavior for an error case.

Reply:
Ops, I mixed this with `BackingField` behavior